test/inductor/expect/CodegenCpuTests.test_upsample_nearest1d_cpu.expect

```diff
@@ -0,0 +1,157 @@
+
+from ctypes import c_void_p, c_long
+import torch
+import math
+import random
+from torch import empty_strided, as_strided, device
+from torch._inductor.codecache import AsyncCompile
+from torch._inductor.select_algorithm import extern_kernels
+
+aten = torch.ops.aten
+assert_size_stride = torch._C._dynamo.guards.assert_size_stride
+async_compile = AsyncCompile()
+
+import triton
+import triton.language as tl
+from torch._inductor.triton_ops.autotune import grid, start_graph, end_graph
+from torch._C import _cuda_getCurrentRawStream as get_cuda_stream
+
+
+kernel_cpp_0 = async_compile.cpp('''
+#include "/tmp/torchinductor_nkaretnikov/ao/caomrguwaptybystzop6sbmdii67jqxwyhyzjizs5gbwosh2mdkg.h"
```

Comment:
Will need to update these paths. Left them for testing purposes.

Will also need to add all other files. Waiting for CI.